mail_sendgrid/wizards/email_template_preview.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2017 Compassion CH (http://www.compassion.ch)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, api
+
+
+class EmailTemplatePreview(models.TransientModel):
+    """ Put the preview inside sendgrid template """
+    _inherit = 'email_template.preview'
+
+    @api.onchange('res_id')
+    @api.multi
+    def on_change_res_id(self):
```

Comment:
`onchange_[...]`

Reply:
No, because the parent method in odoo/addons/mail has this name

Reply:
sorry, my bad, I missed the `super` call ;P